### PR TITLE
Canvas sig color

### PR DESCRIPTION
**Pull request by calvinhfcheng**
Let's try dis out

https://wealthsimple.atlassian.net/browse/FUND-2327

### CODE REVIEW

**Comment by benjyblack** [comment]
I'm not certain you're right about that @cabello . Isn't the signature saved as an image?

**Comment by cabello** [comment]
@benjyblack yes you are correct we also auto-adopt a signature using a font which I updated the colour here https://github.com/wealthsimple/ares/pull/806

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "lint:js": "eslint 'src'",
     "test": "karma start --singleRun true",
     "test:watch": "karma start --singleRun false",
-    "security": "retire --ignore dist,docs",
+    "security": "retire --package --ignore dist,docs",
```
Comment (benjyblack) [inline package.json]: What is this for? 🤔

Comment (cabello) [reply package.json]: https://github.com/wealthsimple/ws-directives/pull/231/commits/986cbaa9d84c28f582d6907b28eaa06bead15fe8

**Pull request merged**